app/test_main.py

```diff
@@ -1,3 +1,56 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalsYearsTransfer:
+    @pytest.mark.parametrize(
+        "cat_years,dog_years,expected_results",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return [0, 0] if animals ages are 0's too."
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return [0, 0] if animals ages are less then 15"
```

Comment:
The test case description 'should return [0, 0] if animals ages are less then 15' contains a typo. It should be 'less than 15'.